docs/src/man/data.md

```diff
@@ -0,0 +1,133 @@
+# Data
+
+```@meta
+DocTestSetup = quote
+    using PGFPlotsX
+end
+```
+
+There are multiple ways of representing data in PGFPlots.
+
+## [Table and TableData](@id table_header)
+
+A `Table` represents a matrix of data where each column is labeled. It can simply point to an external data file or store the data inline in the `.tex` file. `Table`s can have options.
+
+`TableData` is the representation of just the data, without the `table[options]` part. It is useful for inline tables in specials cases. Also, calls to `Table` use `TableData` to convert the arguments.
```

Comment:
The last sentence here is a bit confusing to me. How does it help the reader to know this?